tests/manifest.ttl

```diff
@@ -1,17 +1,27 @@
-## Distributed under both the "W3C Test Suite License" [1]
-## and the "W3C 3-clause BSD License".
 ## [1] https://www.w3.org/Consortium/Legal/2008/04-testsuite-license
 ## [2] https://www.w3.org/Consortium/Legal/2008/03-bsd-license
 
-PREFIX rdf:    <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
-PREFIX rdfs:   <http://www.w3.org/2000/01/rdf-schema#>
-PREFIX mf:     <http://www.w3.org/2001/sw/DataAccess/tests/test-manifest#>
-PREFIX rdft:   <http://www.w3.org/ns/rdftest#>
-PREFIX trs:    <https://w3c.github.io/rdf-star/tests#>
+@PREFIX rdf:    <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@PREFIX rdfs:   <http://www.w3.org/2000/01/rdf-schema#> .
+@PREFIX mf:     <http://www.w3.org/2001/sw/DataAccess/tests/test-manifest#> .
+@PREFIX rdft:   <http://www.w3.org/ns/rdftest#> .
+@PREFIX trs:    <https://w3c.github.io/rdf-star/tests#> .
+@PREFIX dct: <http://purl.org/dc/terms/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix foaf: <http://xmlns.com/foaf/0.1/> .
 
 trs:manifest  rdf:type mf:Manifest ;
-    rdfs:label "RDF-star test suite" ;
-    rdfs:seeAlso <https://w3c.github.io/rdf-tests/> ;
+  rdfs:label "RDF-star test suite"@en ;
+	rdfs:label "La suite des tests pour RDF-star"@fr;
+	rdfs:label "Conjuto de pruebas para RDF-star"@es;
+	dct:issued "2021-06-21"^^xsd:date ; #not sure if this was the first release date
+	dct:creator [ foaf:name "Gregg Kellog" ] ; #maybe add foaf:homepage
```

Comment:
I don't think we should assert who "created" the suite - the important point is the community owns and maintains the work (much like the trend not to have "author" in open source code).

Reply:
In any case, I'm not the sole creator. It could go on each test, but that would be quite a burden. Date can be useful, if it is properly maintained, but it should be replicated to the JSON-LD and HTML versions of the manifest, as well (along with other bits) to keep the different versions of the test manifests isomorphic.

Reply:
> In any case, I'm not the sole creator. It could go on each test, but that would be quite a burden. Date can be useful, if it is properly maintained, but it should be replicated to the JSON-LD and HTML versions of the manifest, as well (along with other bits) to keep the different versions of the test manifests isomorphic.

Re creator, I can just put the community group. I agree that the date should be replicated in all the files. We are not having many modifications for this set of manifest files. I can't imagine more than 3 modifications/updates in the lifetime of a WG. I'm in favour of properly publish such assets with proper metadata.

Reply:
> I don't think we should assert who "created" the suite - the important point is the community owns and maintains the work (much like the trend not to have "author" in open source code).

Well, the same can be applied to the HTML document. We have editors, authors and contributors. Why don't we also add them in the RDF files we are publishing?